content/en/docs/concepts/services-networking/dns-pod-service.md

```diff
@@ -18,19 +17,46 @@ Kubernetes DNS schedules a DNS Pod and Service on the cluster, and configures
 the kubelets to tell individual containers to use the DNS Service's IP to
 resolve DNS names.
 
-### What things get DNS names?
-
 Every Service defined in the cluster (including the DNS server itself) is
-assigned a DNS name.  By default, a client Pod's DNS search list will
-include the Pod's own namespace and the cluster's default domain.  This is best
-illustrated by example:
+assigned a DNS name. By default, a client Pod's DNS search list will
+include the Pod's own namespace and the cluster's default domain. 
+
+### Namespaces of Services 
+
+DNS queries for services change based on namespace of the pod. DNS queries
+that don't specify a namespace are limited to the pod's namespace. Access
+services in other namespaces by including the namespace in the DNS query. 
+
+For example, consider a pod in a `test` namespace. A `data` service is in 
+the `prod` namespace. 
+
+A query for `data` returns no results, because it uses the pod's `test` namespace. 
+
+A query for `data.prod` returns the intended result, because it specifies the namespace. 
+
+DNS queries may be expanded using the pod's `/etc/resolv.conf`. Kubelet 
+sets this file for each pod. For example, A query for just `data` may be 
+expanded to `data.test.cluster.local`. The values of the `search` option 
+are used to expand queries. [Learn more](https://www.man7.org/linux/man-pages/man5/resolv.conf.5.html) 
```

Comment:
nit: address the reader, You can learn more ... or To learn more about ... see [resolve.conf](...).